PocketKit/Sources/Sync/Operations/FetchList.swift

```diff
@@ -95,7 +100,7 @@ class FetchList: SyncOperation {
 
     private func fetchPage(_ pagination: PaginationSpec) async throws -> GraphQLResult<FetchSavesQuery.Data> {
         let query = FetchSavesQuery(
-            token: token,
+            token: appSession?.session?.accessToken ?? "",
```

Comment:
at this point the token should have a value but because we're passing in the appSession and not token, have to do this check here - let me know if there's a better way!